src/XperienceCommunity.Localization.Base/Localizer/XperienceStringLocalizerFactory.cs

```diff
@@ -0,0 +1,31 @@
+﻿using CMS.ContentEngine;
+using CMS.DataEngine;
+using CMS.Helpers;
+using CMS.Websites.Routing;
+using Microsoft.Extensions.Localization;
+
+namespace XperienceCommunity.Localizer.Internal
```

Comment:
Can you please fix this namespace?

Reply:
Moved to folder so internal now appropriate